src/res/translation/translation_de_DE.ts

```diff
@@ -1048,7 +1050,7 @@ We muted your channel and activated &apos;Mute Myself&apos;. Please solve the fe
     <message>
         <location filename="../../clientdlg.cpp" line="319"/>
         <source>Auto-Adjust all &amp;Faders</source>
-        <translation type="unfinished"></translation>
+        <translation>Selbst-Justage aller &amp;Fader</translation>
```

Comment:
Hrm, I'm wondering if there are more natural descriptions than `Selbst-Justage`.
Thinking out loud:
- `Automatische Fader-Anpassung`
- `Automatisches Einpegeln (aller Fader)` <- This might be the most natural translation, at least for audio-interetested people? (either with braces or without)
- `Automatisches Einstellen aller Fader` <- This would be most literal translation, I'd say?

What do you think?

Reply:
Automatisches Einpegeln (aller Fader)  sounds perfect  !